Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/WalletConnect.cs

```diff
@@ -103,6 +103,8 @@ public NativeWebSocketTransport Transport
 
 		private async void Awake()
 		{
+#if !UNITY_WEBGL
```

Comment:
I think we should control it from outside, not modify the logic of WalletConnect

Reply:
Like create a script that will activate wallet connect if it is a mobile build

Reply:
Just disabling walletconnect game object will do I think.